kubetest2/internal/deployers/eksapi/node.go

```diff
@@ -96,7 +105,63 @@ func (m *NodegroupManager) createNodegroup(infra *Infrastructure, cluster *Clust
 	}
 }
 
-func (m *NodegroupManager) createManagedNodegroup(infra *Infrastructure, cluster *Cluster, opts *deployerOptions) error {
+// createPlaceholderDeployment creates a Deployment with the specified number of replicas that requires
+// each replica to be scheduled on different nodes.
+// This ensures that (at least) the specified number of nodes exist in an EKS Auto cluster
+func (m *nodeManager) createPlaceholderDeployment(k8sClient *kubernetes.Clientset, replicas int) (*appsv1.Deployment, error) {
```

Comment:
Where do these get cleaned up at?

Reply:
I didn't add any cleanup for them, since we can just delete the cluster and this will go down with the ship. Are there any cases that wouldn't work?

Reply:
Ah, yeah that's fine. I was still in static cluster land.